tutorials/vue/vue-websocket.md

```diff
@@ -0,0 +1,47 @@
+[WebSockets](https://masteringjs.io/tutorials/node/websockets) are a great tool for when you want to show real time changes in data.
+For example, displaying stock market changes, or in chat applications which reuse the WebSocket connection for the current users.
```

Comment:
I'd make this a bit more concrete by saying that the server can push stock market price changes to the client, rather than the client asking for changes every 5 seconds.